docs/concept-exercises.md

```diff
@@ -79,6 +79,10 @@ Once the student completes the exercise they will be shown this file, which shou
 
 See the C# floating-point-numbers exercise's [after.md file][csharp-docs-after.md] for an example.
 
+### `.docs/source.md`
```

Comment:
Isn't this document optional?

Reply:
It's only optional if there are no sources. Adding `(optional)` here could be misleading.

Reply:
Well, I feel that the default is that it is optional, it is only included if there are sources, which I expect to be the exception.

Reply:
Does 55d4828 resolve this or I should add an additional `(optional)` here?

Reply:
How about `(required if there are third-party sources)` or something, to make it more explicit about when it's optional or not. No need to use one word if we can use 8 words and add clarity.

Reply:
```suggestion
### `.docs/source.md` (required if there are third-party sources)
```

Reply:
I like it!